src/Microsoft.AspNetCore.Server.Kestrel/Http/MessageBody.cs

```diff
@@ -353,12 +394,32 @@ private static bool TakeChunkedLine(SocketInput baton, ref int chunkSizeOut)
                 }
             }
 
+            private static void ThrowInvalidFormat()
+            {
+                throw new InvalidOperationException("INVALID REQUEST FORMAT");
```

Comment:
Can you switch to lowercase? Not sure why it was all uppercase to begin with, but since you're changing this it would be nice to be consistent (and better looking :smile:) with the rest.